src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -63,19 +63,27 @@ public UserService(UserDAO userDAO, UserPropertyDAO userPropertyDAO, UserRoleDAO
     }
 
     public static class SimplifiedUser {
+        public Integer userId;
+        @Deprecated
         public Integer dacUserId;
         public String displayName;
         public String email;
 
         public SimplifiedUser(User user) {
+            this.userId = user.getUserId();
+            this.dacUserId = user.getUserId();
```

Comment:
Was wondering if retaining the dacUserId here was intentional (lines 68, 74, 87, 88)

Reply:
This is very intentional. Until the UI can rely on only `userId`, we need to continue populating the `dacUserId` value.